app/src/main/java/openfoodfacts/github/scrachx/openfood/views/product/ProductActivity.java

```diff
@@ -275,8 +276,10 @@ private void setupViewPager( ViewPager viewPager )
 			adapterResult.addFragment( new NutritionProductFragment(), menuTitles[2] );
 			if( mState.getProduct().getNutriments() != null && mState.getProduct().getNutriments().contains(Nutriments.CARBON_FOOTPRINT) )
 			{
-				adapterResult.addFragment( new EnvironmentProductFragment(), menuTitles[4] );
-			}
+				adapterResult.addFragment( new EnvironmentProductFragment(), "Environment" );
+			} else if (mState.getProduct().getEnvironmentInfocard() != null && !mState.getProduct().getEnvironmentInfocard().isEmpty()) {
+			    adapterResult.addFragment(new EnvironmentProductFragment(), "Environment");
+            }
```

Comment:
Spacing is a bit off for this closing brace

Reply:
I'll fix this